### PR TITLE
Add reset password

### PR DESCRIPTION
**Pull request by Karkavin**

### CODE REVIEW

File: angio-frontend/src/modules/user/components/ResetPasswordForm.vue [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                        v-bind:disabled="fetching"
+                        type="text"
+                        name="email"
+                        prepend-inner-icon="person"
+                        browser-autocomplete="off"
+                        outline
+                        required
+                ></v-text-field>
+                <BuiltInErrorMessage
+                        v-bind:error-messages="errorMessages.map(message => $t(message))"
```
Comment (nikon-petr) [inline angio-frontend/src/modules/user/components/ResetPasswordForm.vue]: Move to computed: [vue ts computed](https://github.com/vuejs/vue-class-component), [ts class getters](https://www.typescriptlang.org/docs/handbook/classes.html)

File: angio-frontend/src/modules/common/components/BuiltInErrorMessage.vue [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<script lang="ts">
+    import {Component, Prop, Vue} from 'vue-property-decorator';
+
+    @Component
+    export default class BuiltInErrorMessage extends Vue {
+
+        @Prop()
+        public readonly errorMessages!: string[];
+    }
+</script>
```
Comment (nikon-petr) [inline angio-frontend/src/modules/common/components/BuiltInErrorMessage.vue]: Required empty line in the end of file

Comment (Karkavin) [reply angio-frontend/src/modules/common/components/BuiltInErrorMessage.vue]: Done

File: angio-frontend/src/modules/user/services/userApiService.ts [hunk 3]
```diff
@@ -42,4 +42,9 @@ export class UserApiService {
         log.debug('create getSettings request');
         return axios.get<Response<UserSettingsModel>>('/user/me/settings');
     }
+
+    public static resetPassword(email: string): AxiosPromise<void> {
+        log.debug(`reset current user password request with email: ${JSON.stringify(email)}`);
```
[the remark this answers is not in the document]
Comment (Karkavin) [reply angio-frontend/src/modules/user/services/userApiService.ts]: Done

File: angio-frontend/src/modules/user/components/ResetPasswordForm.vue (hunk 1 above)
Comment (Karkavin) [reply angio-frontend/src/modules/user/components/ResetPasswordForm.vue]: Done in: LoginForm, ResetPasswordForm, AccountActivationForm

**Comment by Karkavin** [comment]
Refactoring has already finished

**Pull request merged**